app/views/adoption_applications/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<hr>
+<h1><center>Application # <%= "#{@application.id}" %> </center></h1>
```

Comment:
We can test for any nil edge cases to ensure the website works for all situations